packages/docs/page-config/ui-elements/data-table/examples/StaticSlots.vue

```diff
@@ -108,23 +84,26 @@ export default defineComponent({
     ];
 
     return {
-      items: users,
+      items,
       columns,
     };
   },
 });
 </script>
 
 <style lang="scss" scoped>
-.table-example-slots__head {
+.table-slots__head {
   background-color: var(--va-info);
```

Comment:
It is still hard to read this demo. Let's remove extra text like "CUSTOM CELL WHICH SPAN ALL CELLS " and just show "#headAppend slot" in bold and primary color.